### PR TITLE
Create blank-staging-review-feedback

### LINKED ISSUE #28867: Post-pilot enhancement: Create blank template for Staging Review issues
## Issue Description
In the [Staging Review pilot ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25763), Product Support identified a [series of enhancements](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25762) that should be fixed prior to going live with the new process. This ticket will be for creating a blank github issue template that matches the format of the tickets created from the bulk upload process. If a collab cycle reviewer doesn't get their review in on time, or if a last minute issue is found, using the DraftingTemplate to then bulk upload would be inefficient. This is when the blank github template would be useful.

*Please refer to ticket #28861 as the template layout will be changing.

---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

### PR DESCRIPTION
**Pull request by allison0034**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/blank-staging-review-feedback [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+name: Blank Staging Review Feedback Ticket
```
Comment (shiragoodman) [inline .github/ISSUE_TEMPLATE/blank-staging-review-feedback:2]: you can remove "Blank"

File: .github/ISSUE_TEMPLATE/blank-staging-review-feedback [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+### Platform Issue
+
+
+### Issue Details
+
+
+### Link, screenshot or steps to recreate
+
+
+### VA.gov Experience Standards
```
Comment (shiragoodman) [inline .github/ISSUE_TEMPLATE/blank-staging-review-feedback:32]: this should be singular - Standard*

File: .github/ISSUE_TEMPLATE/blank-staging-review-feedback [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- [ ] Comment on the ticket if there are questions or concerns
+- [ ] VFS team closes the ticket when the issue has been resolved
+
+## Platform directions
+- Update "Issue Title"
+- Link to collab cycle touchpoint meeting epic
+- Add to the Feedback pipeline on the Platform Product Support board
+- Add your feedback
+- Add assignees: VFS PM
+- Add labels including appropriate practice area and if it is launch blocking
```
Comment (shiragoodman) [inline .github/ISSUE_TEMPLATE/blank-staging-review-feedback:54]: For labels we should be more specific. Add:

- VFS team label
- VA.gov Experience Standard Issue number (`CCIssueXX.YY`) where XX is the category number and YY is the issue number, per the [VA.gov Experience Standards ](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)
- Practice area labels (`508/accessibility, IA, content, design`) per the [VA.gov Experience Standards ](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)
- 508 defect label, if applicable (`508-defect-0, 508-defect-1, 508-defect-2, 508-defect-3, 508-defect-4`)
- `launch-blocking` label, if applicable

**Review by shiragoodman — CHANGES_REQUESTED** [review]

**Review by allison0034 — COMMENTED** [review]
Done

**Pull request merged**